float64 pos_x
float64 pos_y
float64 lin_vel_x
float64 ang_vel_z